src/modules/layouts/components/UserMenu/UserMenu.tsx

```diff
@@ -11,66 +11,54 @@ import { useAuth } from '@modules/user-auth/contexts/AuthContext';
 import AuthApi from '@services/AuthApi';
 
 const UserMenu: VFC = () => {
+  const { t } = useTranslation();
+  const router = useRouter();
   const { isAuthenticated } = useAuth();
+  const { registerModal, loginModal } = useModals();
 
   return (
     <Menu button={<UserMenuButton />}>
       {isAuthenticated ? (
-        <AuthenticatedUserMenuContent />
+        <MenuItemGroup>
+          <MenuLink href="/account">
+            {t('common:routes.account.index')}
+          </MenuLink>
+        </MenuItemGroup>
       ) : (
-        <UnauthenticatedUserMenuContent />
+        <MenuItemGroup tw="font-semibold">
+          <MenuItem onSelect={registerModal?.show}>
+            {t('common:userMenu.register')}
+          </MenuItem>
```

Comment:
These are shown as `div`

```suggestion
          <MenuItem asChild onSelect={registerModal?.show}>
            <Button>
            {t('common:userMenu.register')}
            </Button>
          </MenuItem>
```

Reply:
I notice Radix example also uses div, is this really expected behavior.

Reply:
Reach UI also produces `div` as the underlying element. I applied `as="span"` on the previous implementation only because I don't want an item to be a div. A11y-wise there should be no issue since they're provided with all the necessary `aira-*` attributes and roles.

But I'd rather made the change to `span`.